sdk/keyvault/keyvault-keys/test/public/utils/testClient.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { testPollerProperties } from "./recorderUtils";
+import { KeyClient } from "../../../src";
+
+export interface TestClientInterface {
+  client: KeyClient;
+  formatName: (name: string) => string;
+  purgeKey: (keyName: string) => Promise<void>;
+  flushKey: (keyName: string) => Promise<void>;
+}
+export default class TestClient implements TestClientInterface {
```

Comment:
I have a proposal! instead of `formatName` let's just use `recorder.getUniqueName`. And then let's just get rid of calls to `flushKey` and `purgeKey` - I've been doing that to make our tests faster since the KV gets created and deleted anyway. Then we won't need the testClient at all. What do you think?

Reply:
I love that idea! Thank you 🙏 I’ll take it

Reply:
Wait, the main point of this test is to restore a key backup, so the unique name won’t help 🤔 I do the purge to then recover the backup. Let’s sync over teams!

Reply:
We talked over teams. Here’s what I’ll do:

This class is intended to help us do some common functionality on many tests. This specific extension is only used in one test, so we can grab the internals of this extra function and move it to the test directly, then remove this class.

I’ll do that!